### PR TITLE
Script to run Node Vertical with Golang cluster-loader and pbench-ansible 

### PR DESCRIPTION
**Pull request by wabouhamad**
This script will start pbench on an OCP environment already registered for pbench with pbench-ansible, and runs Node Vertical test with GoLang cluster loader. 

At the end of script run, links to pbench results (internal and external) will be provided in script output.

### CODE REVIEW

**Comment by wabouhamad** [comment]
@mffiedler PTAL

File: openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+ls -ltr /root/.kube/config
+cat /root/.kube/config
+
+# Next step need to SCP from master the /opt/pbench-agent/config/pbench-agent.cfg, to support or test instance
+# where we will run the pbench-agent.  This has config details for where to copy pbench results after test is done
+
+echo -e "\nSCP the /opt/pbench-agent/config/pbench-agent.cfg file from Master node ${MASTER_PUBLIC_DNS} to this Test Client host"
+cp /opt/pbench-agent/config/pbench-agent.cfg /opt/pbench-agent/config/pbench-agent.cfg_ORIG
+scp root@${MASTER_PUBLIC_DNS}:/opt/pbench-agent/config/pbench-agent.cfg /opt/pbench-agent/config/pbench-agent.cfg
```
Comment (jeremyeder) [inline openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh]: why doesnt the ansible-host have the pbench config baked in?

File: openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    echo -e "\nssh to $node: "
+    ssh ${node} "pbench-clear-tools; pbench-clear-results"
+  done
+
+  sleep 10
+
+  echo -e "\nSSH to test Client and register pbench on all nodes:"
+
+
+  # pbench register master node first.  Handle case where we have more than one Master later, add for loop.
```
Comment (jeremyeder) [inline openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh]: i thought the registration was done by the pbench ansible job?

Comment (wabouhamad) [reply openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh]: Default is using pbench-ansible for registration.  This is an optional step that allows you to re-register pbench if you want to just run it from master when you don't have a pbench-controller node or test client available.

File: openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh (hunk 1 above)
Comment (wabouhamad) [reply openshift_scalability/ci/scripts/pbench-controller-start-node-vertical.sh]: The ansible-host does have the config baked in, this is only needed if you want to run pbench from master when a test client or pbench-controller host is not available

**Pull request merged**